flink-connector-activemq/pom.xml

```diff
@@ -101,14 +101,14 @@ under the License.
         <dependency>
             <groupId>org.powermock</groupId>
             <artifactId>powermock-module-junit4</artifactId>
-            <version>1.5.5</version>
+            <version>2.0.9</version>
```

Comment:
is this mandatory for flink update?
if not will be better in other commit, only for powermock update (and related)